lib/poller.js

```diff
@@ -13,9 +13,7 @@ function failOnTimeout (timeout, uuid) {
   const t = moment.duration(timeout).as('seconds')
   /* eslint-disable no-throw-literal */
   throw (
-    `User-specified or default time out reached after ${t} seconds.\n` +
-    'Analysis continues on server and may have completed; so run again?\n' +
```

Comment:
The reason I liked the above line is that in general for error messages I keep in mind: what I saw the remediation is, if there is one. 

That said, I don't have strong feelings on this.

Reply:
True, but the remediation may also depend on the client. For example, in an IDE this particular error message might not make sense.

Reply:
But since this is a "throw" the IDE can just ignore it in the "catch".

Reply:
Updated.